cmd/incus/top.go

```diff
@@ -0,0 +1,282 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	incus "github.com/lxc/incus/v6/client"
+	cli "github.com/lxc/incus/v6/internal/cmd"
+	"github.com/lxc/incus/v6/internal/i18n"
+	"github.com/lxc/incus/v6/internal/server/metrics"
+)
+
+type cmdTop struct {
+	global *cmdGlobal
+}
+
+type MetricSet struct {
```

Comment:
Since it's used internally only it doesn't need to be exported, so should just be `metricSet`